### DISCUSSION TITLE
target icon

### DISCUSSION THREAD

**Discussion by RIYA-YADAVV** [tailwindlabs/heroicons#344] [Ideas]
![image](https://user-images.githubusercontent.com/78373992/130963779-3c7f478e-2b78-4d16-be66-b730fa1dc26b.png)
Can you please provide hero icon similar to this image

**Comment by razzeee** [comment]
While I would like target icons, I would hope for it to be more of a archery target, then a crosshair, like suggested here.

**Comment by e-pacheco-div** [comment]
I need something like this as well!!

**Comment by iuioiua** [comment]
I think something like this would be great as a "use my location" button.
